### PR TITLE
User logout menu

### PR DESCRIPTION
**Pull request by alihamoody**
# Description
Added User Logout Menu

Note: we need to revisit the `<Link dark>` component. The font should be white instead of black.
@mbifulco Let me know your thoughts. The `<Link>` component is used here for logout.

### CODE REVIEW

**Comment by mbifulco** [comment]
I just pushed a commit which adds the storybook backgrounds addon to the project.  This allows us to simulate the components on top of a background color which they'll be used with.  With the dark background behind the logout menu ([here](https://deploy-preview-60--gymnasium-storybook.netlify.com/?path=/story/userlogoutmenu--default)), it looks like we have a couple issues - the log out link itself gets lost due to its color, and user name has a white background.

As far as the `dark` prop goes, I think it might make sense to refactor the name of that prop, and add another:

- `dark` should become `isDark`: this should indicate that the link itself should have a dark presentation, and turn orange on hover
- a second prop named `isLight` should be added, which will make the link appear with a white color, and turn orange on hover
- the default behavior should remain if neither `isLight` or `isDark` is applied: the link will be orange, and have an underline on hover

**Comment by alihamoody** [comment]
Fixed background color.
Moving forward to the links issue.

**Comment by mbifulco** [comment]
Thanks!

File: src/components/Link/Link.tsx [hunk 1]
```diff
@@ -2,45 +2,54 @@
...
   children,
   to,
-  dark = false,
+  isDark = false,
+  isLight = false,
   ...rest
 }: Props): React.ReactElement => (
-  <a css={dark ? styleDark : styleLight} href={to} {...rest}>
+  <a
+    css={[baseStyle, isDark && darkStyle, isLight && lightStyle]}
```
Comment (mbifulco) [inline src/components/Link/Link.tsx]: Really cool that these take an array - much nicer than having to do a deep merge on an object!

Comment (alihamoody) [reply src/components/Link/Link.tsx]: Agreed. Very Cool.

**Pull request merged**